blog/JBTIS-422-Final.adoc

```diff
@@ -0,0 +1,169 @@
+= JBoss Tools Integration Stack - Integration tooling for Fuse, BRMS, and Data Virt + Early Access Data Transformation, SAP tooling and SwitchYard
```

Comment:
okey - your titles are getting longer and longer breaking all kind of formatting when aggregated ;)

Can you please come up with a *smaller* title ? like "Integration Stack for Kepler now with Fuse and SAP support" or similar/shorter :)